build/scripts/do-release-candidate.sh

```diff
@@ -1,20 +1,17 @@
 #!/bin/sh -ex
 
-if [ -z "${CIRCLE_PULL_REQUEST}" ] && [ -n "${CIRCLE_TAG}" ] && [ "${CIRCLE_PROJECT_USERNAME}" = "weaveworks" ] ; then
-  export RELEASE_DESCRIPTION="${CIRCLE_TAG}"
-  RELEASE_NOTES_FILE="docs/release_notes/${CIRCLE_TAG/-rc.*}.md"
+if [ -z "${GITHUB_REF_NAME}" ] || [ "${GITHUB_REF_TYPE}" != "tag" ] ; then
+  echo "Expected a tag push event, skipping release workflow"
+  exit 1
+fi
 
-  if [[ ! -f "${RELEASE_NOTES_FILE}" ]]; then
-    echo "Release notes ${RELEASE_NOTES_FILE} not found. Exiting..."
-    exit 1
-  fi
+RELEASE_NOTES_FILE="docs/release_notes/${GITHUB_REF_NAME/-rc.*}.md"
```

Comment:
we have not been appending `-rc` to the release notes, is it necessary? it looks like line 10 will fail if the file name doesn't match this pattern..

Reply:
Yes, it's necessary. We don't append `-rc` to the release notes file but we do for RC tags. `GITHUB_REF_NAME` for RCs will have the `-rc.*` suffix and this line removes that suffix to get the release notes filename.

Reply:
Thanks for the explanation! 👍